wrappers/kn/README.md

```diff
@@ -0,0 +1,59 @@
+# ZXing-C++ Kotlin/Native Library
+
+## Install
+
+The easiest way to use the library is to fetch if from _mavenCentral_. Simply add
+
+```gradle
+implementation("io.github.zxing-cpp:kotlin-native:2.2.1")
+```
+
+to your `build.gradle.kts` file in the `dependencies` section of `nativeMain` source set.
+
+## Use
+
+A trivial use case looks like this:
+
+```kotlin
+import zxingcpp.BarcodeFormat.EAN13
+import zxingcpp.BarcodeFormat.QRCode
+import zxingcpp.BarcodeReader
+import zxingcpp.ImageFormat
+import zxingcpp.ImageView
+
+
+val barcodeReader = BarcodeReader().apply {
+   formats = setOf(EAN13, QRCode)
+   tryHarder = true
+   maxNumberOfSymbols = 3
+   // more options, see documentation
+}
+
+@OptIn(ExperimentalUnsignedTypes::class)
+fun process(
+   data: UByteArray,
+   width: Int,
+   height: Int,
+   imageFormat: ImageFormat = ImageFormat.Lum // ImageFormat.Lum assumes grey scale image data.
+) {
+   ImageView(data, width, height, imageFormat)
+   barcodeReader.read(image).joinToString("\n") { barcode: Barcode ->
```

Comment:
`image` is an unknown/undefined symbol here

Reply:
fixed in: https://github.com/zxing-cpp/zxing-cpp/pull/719/commits/e58791bbb9fe4e71f8f28268243a91e9df43e6db